toast-tk-plugin-loader/src/main/java/io/toast/tk/plugin/AgentBoot.java

```diff
@@ -0,0 +1,42 @@
+package io.toast.tk.plugin;
+
+import java.lang.instrument.Instrumentation;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.toast.tk.agent.config.AgentConfigProvider;
+
+public class AgentBoot {
+
+	private static final Logger LOG = LoggerFactory.getLogger(AgentBoot.class);
+
+	private static Instrumentation instrumentation;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "instrumentation" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)